src/gui/folderman.cpp

```diff
@@ -1495,4 +1497,12 @@ bool FolderMan::prepareFolder(const QString &folder)
     return true;
 }
 
+void OCC::FolderMan::reenableSyncAfterFolderAdded()
+{
+    // The vfs folder might be set up async, schedule it after enough time to setup
```

Comment:
A timer is not really a guarantee... is there any way we can ensure that this is being executed in the right order?